examples/malfunction-indicator-light/mail-notification/build.gradle

```diff
@@ -14,9 +14,9 @@ apply plugin: 'io.spring.dependency-management'
 
 bootJar {
     baseName = 'mail-notification'
-    version = '0.2.0'
 }
 
+version = '0.1.0'
```

Comment:
I'm sorry for checking the Jenkins build just now! This change causes the build to fail: The [Jenkinsfile](https://github.com/eclipse/kuksa.cloud/blob/master/Jenkinsfile) still attempts to copy version 0.2.0 of mail-notification. Could you adapt the Jenkinsfile as part of this PR.

Jenkins will by default likely continue use the Jenkinsfile from the master branch so to test a Jenkinsfile from the PR you will have to replay a previous run and paste the Jenkinsfile from the PR into the replay dialogue.